woocommerce/woocommerce-ios
# Refunds in Order Details: aggregate data and item sorting

Closes #1731 and Closes #1717 

Ref. p99K0U-2no-p2

This PR calculates the total quantities remaining for the listed order items and also re-calculates the order item totals based on the new quantity. The order items are sorted by productID and variationID, in ascending order. The Refunded Products screen is also sorted the same way.

### Screenshots
**Before**
<img src="https://user-images.githubusercontent.com/1062444/72479538-5358e180-37ba-11ea-8da1-d7e66387fef4.png" width="350" />

**After**
<img src="https://user-images.githubusercontent.com/1062444/72479568-6ff51980-37ba-11ea-999b-12bc616f56c3.png" width="350" />


### To test
The goal is to create multiple refunds with several of the same product type(s) in the refunds.

1. On your test site, have an order with multiple product types. Make sure to note the Order Number.
2. Find the order and "refund by item".
3. Make another refund. Refund a couple more of the same product, and add another product to make it interesting.
4. Make another refund. Refund a couple more of the same product(s) or refund a different item by itself.
5. Check out the develop branch and navigate in the app to: Orders tab > Order Number (Order Details screen). 
6. Check out this branch. Navigate to the same location. The cells should now be summarized.


Update release notes:
